nservicebus/messaging/command-event-comparison-table.include.md

```diff
@@ -0,0 +1,9 @@
+|   | Commands | Events |
+|---|:--------:|:------:|
+| Marker Interface | `ICommand` | `IEvent` |
+| Logical Senders | Many | 1 |
```

Comment:
Are logical senders and receivers new defined terms? 
A logical sender or a logical receiver can publish an event. From that perspective, this is confusing.